__tests__/app.test.js

```diff
@@ -101,3 +101,33 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("returns 200 status code and an array of articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(response.status).toBe(200);
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles.length).toBe(13);
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
```

Comment:
This is a great use of `jest sorted`, I wonder if it's a lot going on in one test and this assertion for the sort order might be better off in it's own separate test